book/line_editor.md

```diff
@@ -1,124 +1,212 @@
 # Reedline, Nu's Line Editor
 
-Nushell's line editor [Reedline](https://github.com/nushell/reedline) is a
-cross-platform line reader designed to be modular and flexible. The engine is
-in charge of controlling the command history, validations, completions, hints
-and screen paint.
+Nushell's line-editor [Reedline](https://github.com/nushell/reedline) is
+cross-platform and designed to be modular and flexible. The line-editor is
+in charge of controlling the command history, validations, completions, hints,
+screen paint, and more.
 
-## Configuration
+[[toc]]
 
-### Editing Mode
+## Multi-line Editing
 
-Reedline allows you to edit text using two modes: vi and emacs. If not
-specified, the default edit mode is emacs mode. In order to select your
-favorite you need to modify your config file and write down your preferred
-mode.
+Reedline allows Nushell commandlines to extend across multiple lines. This can be accomplished using several methods:
 
-For example:
+1. Pressing <kbd>Enter</kbd> when a bracketed expression is open.
+
+   For example:
+
+   ```nu
+   def my-command [] {
+   ```
+
+   Pressing <kbd>Enter </kbd> after the open-bracket will insert a newline. This will also occur with opening (and valid) `(` and `[` expressions.
+
+   This is commonly used to create blocks and closures (as above), but also list, record, and table literals:
+
+   ```nu
+   let file = {
+     name: 'repos.sqlite'
+     hash: 'b939a3fa4ca011ca1aa3548420e78cee'
+     version: '1.4.2'
+   }
+   ```
+
+   It can even be used to continue a single command across multiple lines:
+
+   ```nu
+   (
+     ffmpeg
+     -i input.mp4
+     -vf "scale=1280:720,setsar=1:1"
+     -b:v 1500k
+     -preset veryfast
+     -crf 23
+     -c:a aac
+     -b:a 192k
+     -movflags +faststart
+     -y
+     output.mp4
```

Comment:
I think this example is taking up too much screen real estate and in `ffmpeg` being unfamiliar risks confusion. It suffices to show that you can wrap flags and positionals for an external without needing `\`

Reply:
Agreed - I started with it in a `::: details` block to hide it, but talked myself into it being "okay".

I've replaced it with a `tar` example which is much shorter and also placed it behind `::: details Example` for flow.